src/lib/bcmp/bm/bm_util.h

```diff
@@ -19,15 +19,15 @@ extern const ip6_addr_t multicast_ll_addr;
   \param *ip address to extract node id from
   \return node id
 */
-static inline uint64_t ip_to_nodeid(void *ip) {
```

Comment:
now defined in `bm_common/util.h`, which has been updated with the latest bm_core update.